napari/__init__.pyi

```diff
@@ -7,18 +7,18 @@ import napari.viewer
 __version__: str
 
 notification_manager: napari.utils.notifications.NotificationManager
-Viewer = napari.viewer.Viewer
-current_viewer = napari.viewer.current_viewer
+Viewer = ...
+current_viewer = ...
 
-gui_qt = napari._qt.qt_event_loop.gui_qt
-run = napari._qt.qt_event_loop.run
-save_layers = napari.plugins.io.save_layers
+gui_qt = ...
+run = ...
+save_layers = ...
 
-view_image = napari.view_layers.view_image
-view_labels = napari.view_layers.view_labels
-view_path = napari.view_layers.view_path
-view_points = napari.view_layers.view_points
-view_shapes = napari.view_layers.view_shapes
-view_surface = napari.view_layers.view_surface
-view_tracks = napari.view_layers.view_tracks
-view_vectors = napari.view_layers.view_vectors
+view_image = ...
+view_labels = ...
+view_path = ...
+view_points = ...
+view_shapes = ...
+view_surface = ...
+view_tracks = ...
+view_vectors = ...
```

Comment:
this changes are caused by https://beta.ruff.rs/docs/rules/#flake8-pyi-pyi PYI015 that is implementation of rule Y015 form https://pypi.org/project/flake8-pyi/

At this moment I do not understand the motivation for this change. But will try to investigate this. 

But in general, I think this change should be reverted, and maybe just rules skipped.

Reply:
Does not seem like a bad rule to me in general, I just don't understand the fix O.o

Reply:
I just use implicit reexport https://mypy.readthedocs.io/en/stable/config_file.html#confval-implicit_reexport that is treated as valid by mypy so do not changed by ruff 

It was also mentioned here https://scientific-python.org/specs/spec-0001/#type-checkers

It was split on multiple lines because ruff configuration [`combine-as-imports`](https://beta.ruff.rs/docs/settings/#combine-as-imports) is set by default to false.